gma/integration_test/src/integration_test.cc

```diff
@@ -340,6 +350,28 @@ void FirebaseGmaPreInitializationTests::SetUpTestSuite() {
 
 // Test cases below.
 
+TEST_F(FirebaseGmaMinimalTest, TestInitializeGmaWithoutFirebase) {
+  // Don't initialize mediation in this test so that a later test can still
+  // verify that mediation has not been initialized.
+  firebase::gma::DisableMediationInitialization();
+  LogDebug("Initializing GMA without a Firebase App.");
+  firebase::InitResult result;
+#if defined(ANDROID)
+  ::firebase::gma::Initialize(app_framework::GetJniEnv(),
+                              app_framework::GetActivity(), &result);
+#else   // !defined(ANDROID)
+  ::firebase::gma::Initialize(&result);
+#endif  // defined(ANDROID)
+  EXPECT_EQ(result, ::firebase::kInitResultSuccess);
+  auto initialize_future = firebase::gma::InitializeLastResult();
+  WaitForCompletion(initialize_future, "gma::Initialize");
+  EXPECT_EQ(initialize_future.error(), 0) << initialize_future.error_message();
```

Comment:
One last nit, I believe WaitForCompletion actually already checks that the error is 0 (unless you pass in a different error to expect).

Reply:
True, done.